src/styles/molecules/_molecules.tag.scss

```diff
@@ -35,24 +63,59 @@
  * -------------------------------------------------------------------
  */
 
+.m-tag__icon,
+.m-tag__button {
+  color: var(--tag-color);
+  transition: color var(--animation-normal);
+
+  .ai {
+    font-size: .75rem;
+  }
+}
+
 .m-tag__label {
   @extend %u-text-truncate;
 
-  display: block;
+  color: var(--tag-color);
+  font-size: .875rem;
   font-weight: var(--font-weight-bold);
+  line-height: normal;
+  margin: 0 var(--spacer-s) 0 var(--spacer-3xs);
   max-width: calc(var(--spacer) * 10);
-  padding: 0 var(--spacer-xs);
+  transition: color var(--animation-normal);
 }
 
-.m-tag__input {
-  @include input-placeholder {
-    color: var(--tag-placeholder-color);
-  }
-
-  background-color: transparent;
+.m-tag__button {
+  background: transparent;
+  border-radius: 2rem;
```

Comment:
Misschien beter in variabele steken?

Reply:
![Screenshot 2021-12-15 at 08 10 06](https://user-images.githubusercontent.com/4232875/146140103-527a8fd1-6e05-44af-97fb-0f3de4ce7068.png)

Ik zou het zeker in een variable kunnen steken, maar weet alleen niet of het super nuttig is. 🙂 

In de afbeelding hierboven heb ik even die transparante sluitknop, even ter verduidelijking, rood gemaakt. Die sluitknop is 2rem groot en ik heb die een border-radius van 2rem gegeven om die volledig rond te maken zodat zijn hoeken (rechtsboven en rechtsonder) niet buiten de tag zouden komen. Die border-radius ga je nooit effectief zien, want die knop is transparant. Dus daarom leek het mij niet super nuttig om die border-radius in een variable te steken, aangezien het visueel nooit gaat opvallen. 🙂

Reply:
Ok, ik snap het. Heeft het dan toch geen zin om er 50% van te maken i.p.v. een vast getal?

Reply:
Hmm ja, ik heb al gemerkt dat border-radius: 50% niet altijd overal goed werkt. Dat de component in kwestie dan niet perfect rond is. Als je effectief de hoogte van de component als border-radius gebruikt, dan werkt het beter. 🙂

Reply:
Ja, excuses, ik bedoelde eigenlijk 100%. Werkt die ook niet overal naar behoren?